kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/ResourceGeneratorBuilder.java

```diff
@@ -0,0 +1,44 @@
+package org.arquillian.cube.kubernetes;
+
+import java.nio.file.Path;
+import org.jboss.shrinkwrap.resolver.api.maven.embedded.BuiltProject;
+import org.jboss.shrinkwrap.resolver.api.maven.embedded.EmbeddedMaven;
+
+public class ResourceGeneratorBuilder {
```

Comment:
Should it really belong to `kubernetes` module?

Reply:
I was thinking the same, but not sure should we create separate module for single class.